third_party/libunwindstack/PeCoffInterface.cpp

```diff
@@ -372,6 +374,23 @@ bool PeCoffInterface<AddressType>::InitSections() {
   return true;
 }
 
+template <typename AddressType>
+bool PeCoffInterface<AddressType>::InitDebugFrameSection() {
+  CHECK(debug_frame_section_data_.has_value());
+
+  debug_frame_.reset(new DwarfDebugFrame<uint64_t>(memory_));
```

Comment:
This looks suspicious. Did you mean `new DwarfDebugFrame<AddressType>(...`

Reply:
Yes, thanks for spotting this!

Reply:
Done.